README.md

```diff
@@ -15,7 +15,7 @@ Requirements
 - python (2.7, 3.5, 3.6)
 - django (1.11, 2.0, 2.1)
 - djangorestframework (>=3.1, <4.0)
-- social-auth-core (>=1.7.0, <2.0)
+- social-auth-core (>=1.7.0, <=3.0)
```

Comment:
Let's write it as `(>= 3.0, <4.0)`.

Reply:
Making this update in the latest commit.